### ISSUE TITLE
[BUG] Using no-custom-classname with DaisyUI

### ISSUE THREAD

**Issue by oscartbeaumont** [francoismassart/eslint-plugin-tailwindcss#129]
**Describe the bug**
I am using [TailwindCSS](https://tailwindcss.com) with the [DaisyUI](https://daisyui.com) plugin and am having issues with the `no-custom-classname` rule. This plugin will report DaisyUI's classes as a custom classname and create an eslint warning.

**To Reproduce**
1. Go to [here](https://stackblitz.com/edit/vitejs-vite-spjry7)
2. Run `yarn lint` in terminal.
3. Observe:
```bash
/home/projects/vitejs-vite-spjry7/src/main.tsx
  7:9   warning  Classname 'not-valid' is not a Tailwind CSS class!  tailwindcss/no-custom-classname # This one is expected
  8:13  warning  Classname 'btn' is not a Tailwind CSS class!        tailwindcss/no-custom-classname # This is a DaisyUI class and shouldn't cause a warning
```

**Expected behavior**
Classes from TailwindCSS plugins such as DaisyUI should not be reported by `tailwindcss/no-custom-classname` as a custom classname. This seems like a reasonable behaviour as this eslint plugin has access to the Tailwind configuration.

Hence in the example repository, there should end up being only one eslint error for the class `not-valid`.

**Comment by francoismassart** [comment]
Hi @oscartbeaumont,
I did take a look at this and it should be fixed soon but I have a question about the classname `not-valid`: is it part of daisyui ? Because I made a quick test https://play.tailwindcss.com/VZjg8NoOez and it looks like it is not.

Thank you for your reply

**Comment by oscartbeaumont** [comment]
`not-valid` is **not** a class of Tailwind or DaisyUI. I was just using it to show a test case where the plugin acts as expected, and is still able to throw errors for unknown classes. Sorry for the confusion.

Thanks for looking into it. I am looking forward for a fix!

**Comment by francoismassart** [comment]
I just published a beta version with the fixes listed on #132 
https://github.com/francoismassart/eslint-plugin-tailwindcss/tree/fix/no-custom-classname-plugin-support#readme

`npm i eslint-plugin-tailwindcss@3.6.0-beta.0`

Please give it a try and provide feedback and or issues 👍 
Thank you

**Comment by francoismassart** [comment]
👋 Any feedback on the latest beta @oscartbeaumont  ?

**Comment by francoismassart** [comment]
Should be fixed in the [latest release `3.6.0`](https://github.com/francoismassart/eslint-plugin-tailwindcss/releases/tag/v3.6.0)

Make sure to:

1. Update the `eslint-plugin-tailwindcss` package
2. Restart your editor
3. Report any issue on this repo
